src/utils/event-utils.ts

```diff
@@ -267,7 +117,7 @@ export const _info = {
         if (!userAgent) {
             return {}
         }
-        const { os_name, os_version } = _info.os(userAgent)
+        const [os_name, os_version] = _info.os(userAgent)
```

Comment:
so for example here when `os_name` and `os_version` were object keys they can't be minified either where they were written multiple times in `_info.os` or here where they're used

this way the string `os_name` isn't even present in the minified output